crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -646,6 +700,27 @@ mod tests {
 		cli.verify()
 	}
 
+	#[tokio::test]
+	async fn user_cancel_send_extrinsic_from_call_data_works() -> Result<()> {
+		let api = set_up_api("wss://rpc1.paseo.popnetwork.xyz").await?;
```

Comment:
```suggestion
		let client = set_up_client("wss://rpc1.paseo.popnetwork.xyz").await?;
```
See above for justification. I assume that this is in the base PR, so happy to adjust it there instead and just rebase.